### ISSUE TITLE
Generators generate JS after install

### ISSUE THREAD

**Issue by alexspeller** [alexspeller/ember-cli-coffees6#15]

**Comment by movstox** [comment]
After install:

```
# If you already have ember-cli-coffeescript installed
npm uninstall ember-cli-coffeescript --save-dev

ember install:addon ember-cli-coffees6
```

generators do this:

```
$ ember g controller me-now
version: 1.13.7
installing controller
  create app/controllers/me-now.js
installing controller-test
  create tests/unit/controllers/me-now-test.js

```

`me-now.js`:
```
import Ember from 'ember';     
        
export default Ember.Controller.extend({
});
```

**Comment by alexphys** [comment]
Same problem here

**Comment by alexspeller** [comment]
This should be resolved with version `0.3.0` - note you now have to install ember-cli-coffeescript as well in your project
